app/assets/stylesheets/modules/header_navbar.scss

```diff
@@ -0,0 +1,58 @@
+.bg-cardinal-red {
+  background-color: $cardinal-red;
+}
+
+#header-navbar {
+  box-shadow: 0 4px 8px -2px $sul-shadow-color;
+  min-height: 100px;
+
+  h1 {
+    a {
+      color: $white;
+      text-decoration: none;
+
+      &:hover {
+        border-bottom: transparent;
+      }
+    }
+  }
+
+  .navbar-brand {
```

Comment:
I _think_ this is technically unused right now.  Not sure if we should use a `navbar-brand` or not.  Can 🔪 this if we think we will never want it.